functests/utils/images/images.go

```diff
@@ -0,0 +1,54 @@
+package images
+
+import (
+	"fmt"
+	"os"
+
+	gomega "github.com/onsi/gomega"
+)
+
+var registry string
+var images map[string]imageLocation
+
+const (
+	// TestUtils is the image name to be used to retrieve the test utils image
+	TestUtils = "testutils"
+	// Stresser is the image name to be used to retrieve the stresser image
+	Stresser = "stresser"
+	// SctpTester is the image name to be used to retrieve the sctptester image
+	SctpTester = "sctptester"
+)
+
+func init() {
+	registry = os.Getenv("IMAGE_REGISTRY")
+
+	images = map[string]imageLocation{
```

Comment:
should you add here also the stresser?

Reply:
Nope, we don't use the stresser in this suite. I will remove the constant too.